### PR TITLE
Update free-courses-ar.md

### PR DESCRIPTION
**Pull request by Adamkaram**
some useful courses from Bigdata channel

## What does this PR do?
adding some useful courses from Bigdata channel

### Description
a free youtube courses in arabic and i'm personally has learn a lot from them 

### Why is this valuable (or not)?
most of arab knew that this channel it's the best resource for complicated topic 

### How do we know it's really free?
youtube video 

### For book lists, is it a book? For course lists, is it a course? etc.
courses

## Checklist:
- [✓ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ✓] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ✓] Include author(s) and platform where appropriate.
- [✓ ] Put lists in alphabetical order, correct spacing.
- [ ✓] Add needed indications (PDF, access notes, under construction).
- [✓] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Adamkaram** [comment]
> Kindly read the contributing guidelines and fix all the linter errors.

done

**Comment by Ajay-singh1** [comment]
@Adamkaram Bro!You have given the individual link of the video and we need playlist link.There you go.

**Comment by Adamkaram** [comment]
> @Adamkaram Bro!You have given the individual link of the video and we need playlist link.There you go.

it's a bunch of video  compressed to one video is this acceptable?

File: courses/free-courses-ar.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
     * [SQL](#sql)
 * [Deep Learning](#deep-learning)
 * [DevOps](#devops)
+* [Docker](#Docker)
```
Comment (eshellman) [inline courses/free-courses-ar.md:19]:
```suggestion
* [Docker](#docker)
```

**Review by eshellman — CHANGES_REQUESTED** [review]
videos are long enough, OK, but please let us know in the PR description to avoid delays

**Comment by Adamkaram** [comment]
done

File: courses/free-courses-ar.md [hunk 2]
```diff
@@ -167,6 +168,7 @@
 * [Learn MySQL&rlm;](https://www.youtube.com/playlist?list=PLDoPjvoNmBAz6DT8SzQ1CODJTH-NIA7R9) - Elzero Web School
 * [MS SQL Server For Beginners](https://www.youtube.com/playlist?list=PL1DUmTEdeA6J6oDLTveTt4Z7E5qEfFluE) - محمد الدسوقى
 * [MySQL tutorials \|\| بناء قواعد البيانات بكفاءة عالية](https://www.youtube.com/playlist?list=PLF8OvnCBlEY25O_Ql0CrgQUAc5NVYkWF2) - TheNewBaghdad
+* [SQL for Data Analysis | شاهد كيف أصبح الفيل والدرفيل أصدقاء](https://www.youtube.com/watch?v=kb-_GbpH3sQ) -  Big Data
```
Comment (eshellman) [inline courses/free-courses-ar.md:171]:
```suggestion
* [SQL for Data Analysis \| شاهد كيف أصبح الفيل والدرفيل أصدقاء](https://www.youtube.com/watch?v=kb-_GbpH3sQ) -  Big Data
```

**Review by eshellman — CHANGES_REQUESTED** [review]

**Pull request merged**